journal.py

```diff
@@ -39,11 +45,26 @@ def write(cls, title=None, text=None, session=None):
         return instance
 
     @classmethod
+    def change(cls, title=None, eid=None, text=None, session=None):
+        if session is None:
+            session = DBSession
+        instance = cls.one(eid)
+        instance.title = title
+        instance.text = text
+        session.add(instance)
+        return instance
```

Comment:
Also, think about the ordering of your arguments.  Putting the title first, then the id, and then the text seems pretty arbitrary.  I'd personally expect the ID first, then the text-based parameters, then the session at the end.